src/cegui/CEGUIProjectManager.cpp

```diff
@@ -120,6 +120,7 @@ void CEGUIProjectManager::ensureCEGUIInitialized()
 
     QSurfaceFormat format;
     format.setSamples(0);
+    format.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
I'm not experienced with OpenGL. What effect this has? Is it necessary for OpenGL 3+ only? If so, maybe you should wrap it into #ifdef ?

Reply:
QT seems to recommend to set it if deprecated functionality (like glBegin/glEnd) is used: https://doc.qt.io/qt-5/qsurfaceformat.html#OpenGLContextProfile-enum

Maybe because the default could change in the future or maybe QSurfaceFormat::CoreProfile is the default under Linux (with CoreProfile glBegin/glEnd is not available anymore). Even if it's not the case QT *should* at least print an error early if the CompatibilityProfile is not availabe